src/test/java/mil/dds/anet/test/resources/AttachmentResourceTest.java

```diff
@@ -114,6 +115,7 @@ public void testDeleteAttachment()
     final AttachmentInput testAttachmentInput =
         AttachmentInput.builder().withFileName("testDeleteAttachment.jpg").withMimeType(mimeType)
             .withDescription("a test attachment created by testDeleteAttachment")
+            .withCaption("testCaption")
```

Comment:
Add an assertion below that the caption of the report attachment matches the attachment input's caption.

Reply:
ok,applied